### ISSUE TITLE
I want to deploy it to a web application, I follow the docs, but failed.

### ISSUE THREAD

**Issue by LRQLRQ** [OHIF/Viewers#831]
Hi, I am new here, these days I try to deploy ohif viewers into a web application ,  but it's hard to do it when I follow the docs. 

**Description**

It seem that the docs is a little old.The docs tell me to download the **react** branch, but I can't find the **react** branch. And the docs tell me to input **yarn run build:web**, but this version don't have this command.
I try to use the master branch , go to **platform/viewers** and input **build:viewer** ,and I copy the **dist** folder into a tomcat's webapps.But it doesn't work. I can't see the ohif viewer on the browser.I really have no idea about how to deploy the application.
in short,I have 3 questions:
 * which branch is suit for deploy.
 * which command I shoule type if I want to deploy it.
 * If tomcat could deploy the application?If not, what can?

Thank you for reading here. I will appreciate it if you can give me some advise.

**Comment by dannyrb** [comment]
@LRQLRQ, we've had a few major releases, and one large restructuring since those docs were written. Unfortunately, the items most impacted were those related to file locations and `npm scripts`. You can read more about how/where things now live here:

https://github.com/OHIF/Viewers/issues/768

More than happy to accept even very small behaviors to clean up what is now incorrect. After we have all of our CI updated, and IE + Edge functionality working as expected, rectifying docs are the next highest priority.

**Comment by dannyrb** [comment]
> which branch is suit for deploy.

`master`

> which command I shoule type if I want to deploy it.

It depends on your use case, but most likely `yarn run build` from the repository root to generate the static assets you should deploy.

> If tomcat could deploy the application?If not, what can?

Any web server capable of serving up static assets will suffice.

**Comment by LRQLRQ** [comment]
> > which branch is suit for deploy.
> 
> `master`
> 
> > which command I shoule type if I want to deploy it.
> 
> It depends on your use case, but most likely `yarn run build` from the repository root to generate the static assets you should deploy.
> 
> > If tomcat could deploy the application?If not, what can?
> 
> Any web server capable of serving up static assets will suffice.

Thank you very very much!! I will have a try!

**Comment by dannyrb** [comment]
@LRQLRQ any updates?

**Comment by LRQLRQ** [comment]
@dannyrb **Thank you for still remember me! I was touched by you.** 
I am a little busy these days, just did little updates about viewers. Here are my new questions I met:
1. I tried to follow what you said, but the console shows these errors. I think I probabely ignored some important config files.
![image](https://user-images.githubusercontent.com/34814090/64474175-c9fa2880-d1a3-11e9-867b-ff8923c5dc43.png)
2. I want to build the `Nginx + Image + Archive` and `User Account Control`  which on the docs, and I learned docker to build it, but I meet some problems, I looked for the solution in these issues, and I 
 found my problems are the seem to #785 .   

**My chores have been dealt with, and I will study the documentation and how to deploy the application in the near future.**
**My English is a little poor ,but I really appreciate that! Thanks again!**

**Comment by dannyrb** [comment]
@LRQLRQ, it does look like `default.js` is missing. In the latest version, `master`, this would be `app-config.js`. It should be produced when you run `yarn run build` from the repository root, alongside the rest of the progressive web application's static assets.

`yarn run build` --> `<repo-root>/platform/viewer/dist/**/*`

You can see how the `Dockerfile` in the root of this repository builds that output, and then uses it in an `nginx` image: 

https://github.com/OHIF/Viewers/blob/master/Dockerfile#L60

**Comment by Diegovictorbr** [comment]
I think my problem relates to this. I have a Tomcat folder called `ohif` where i want to deploy `platform/viewer/dist `content. However, OHIF is not able to find its static files. Here is a screenshot of what happens:

![Sem título](https://user-images.githubusercontent.com/19496924/72273401-0bdf2380-3609-11ea-8112-15bf266f71e0.png)

Tomcat runs at http://149.56.20.137:17545 and i want to serve OHIF at http://149.56.20.137:17545/ohif

I am trying to connect OHIF with an Orthanc instance behind a NGINX reverse proxy.

**Comment by dannyrb** [comment]
@Diegovictorbr, can you ask this question in a new issue? This one is closed and older. We tend to not revisit these for new questions.
